pkg/detectors/dockerhub/v1/dockerhub.go

```diff
@@ -104,80 +118,25 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return
 }
 
-func (s Scanner) verifyMatch(ctx context.Context, username string, password string) (bool, map[string]string, error) {
-	payload := strings.NewReader(fmt.Sprintf(`{"username": "%s", "password": "%s"}`, username, password))
-
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, "https://hub.docker.com/v2/users/login", payload)
-	if err != nil {
-		return false, nil, err
+// Helper function to detect if a string looks like a UUID fragment
+func isLikelyUUIDFragment(s string) bool {
```

Comment:
Why not just use `github.com/google/uuid` to parse the string and verify whether it's a valid UUID?